### DISCUSSION TITLE
Dịch object trong Python

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#127] [Terminology] [answered]
Object của Python mình có nên dịch không?

Context:
> In this example, we created a Precision **object**, then we used it like a function, passing it the labels and predictions for the first batch, then for the second batch (note that we could also have passed sample weights).

**Comment by minhduc0711** [comment, marked as answer]
Em thấy cứ dịch là `đối tượng` thôi ạ
